cmd/gen-go-sample/testdata/sample.want

```diff
@@ -0,0 +1,22 @@
+// [START my_value_set]
```

Comment:
missing `package`

Reply:
The `package` statement is inserted in the `commit` function.